### DISCUSSION TITLE
如何插入算法的伪代码

### DISCUSSION THREAD

**Discussion by wang19950602** [TheNetAdmin/zjuthesis#191] [General]
我在论文中插入算法伪代码的时候，遇到了“latex error: environment algorithm undefined”这个错误，并且在给出的论文示例中也未发现有算法伪代码的示例，是否是因为论文的模板中未包含algorithm这个包，若是这样的话应该在哪个文件里添加，谢谢。

**Comment by TheNetAdmin** [comment]
> 若是这样的话应该在哪个文件里添加，谢谢。

`zjuthesis/config/packages.tex`

**Reply by wang19950602** [reply]
好，谢谢您的解答
